doc/source/limitation.rst

```diff
@@ -82,3 +82,13 @@ Avoid reusing the names of persistent tables for temporary tables
 		<https://dev.mysql.com/doc/refman/8.0/en/temporary-table-problems.html>`_
 
 .. include:: .res/replace.opt.txt
+
+As of version 8.0.25, an INPLACE `ALTER TABLE <https://dev.mysql.com/doc/refman/5.7/en/alter-table.html>`__  query takes an internal shared lock on the table during the execution of the query. The ``LOCK=NONE`` clause is no longer allowed for all of the INPLACE ALTER TABLE queries due to this change.
```

Comment:
We have this limitation 8.0.21. So, It is better we mention 8.0.21 instead of 8.0.25.

Reply:
The link is pointing to 5.7 Docs. Shouldn't we change it to 8.0 Docs?

Reply:
Fixed both.